src/main/resources/plugin.yml

```diff
@@ -0,0 +1,23 @@
+name: AdminTools
```

Comment:
Why the move to /resources? Will it still get packaged as part of the jar with it in this directory?

Reply:
It's supposed to be in resources.
They way it was done before was not the proper way.

Reply:
It still gets packaged, since this is the intended directory.

Reply:
What makes /resources/ better or more proper?

Reply:
See the [Gradle docs](https://docs.gradle.org/current/userguide/java_plugin.html) and the [Maven folder structure docs](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
The official [Bukkit sample plugin](https://github.com/Bukkit/SamplePlugin) uses the format.